docs/source/user-guide/flightsql.md

```diff
@@ -70,7 +70,8 @@ cargo build --release --all --features flight-sql
 ## <a name="docker"/> Run Docker Containers
 
 ```shell
-docker-compose up --build
+source dev/build-ballista-docker.sh
```

Comment:
Thanks @iajoiner. I am curious why `source` is required here rather than just running `./dev/build-ballista-docker.sh` ?